subsys/net/lib/lwm2m/lwm2m_rw_json.c

```diff
@@ -0,0 +1,644 @@
+/*
```

Comment:
Yes, I have a TODO marked in the header:
"Research using Zephyr JSON lib for json_next_token()"